e2e/awslogs_test.go

```diff
@@ -19,25 +19,35 @@ const (
 	awslogsRegionKey              = "--awslogs-region"
 	awslogsStreamKey              = "--awslogs-stream"
 	awslogsGroupKey               = "--awslogs-group"
+	awslogEndpointKey             = "--awslogs-endpoint"
 	testEcsLocalEndpointPort      = "51679"
 	testAwslogsCredentialEndpoint = ":" + testEcsLocalEndpointPort + "/creds"
-	testAwslogsRegion             = "us-west-2"
-	testAwsLogsStream             = "test-stream"
-	testAwsLogsGroup              = "test-shim-logger"
+	testAwslogsRegion             = "us-east-1"
```

Comment:
us-east-1 is default localstack region?

Reply:
Yes. I didn't find explicit word to say it in local stack documentation. But it is included in the example [code](https://docs.localstack.cloud/user-guide/integrations/sdks/go/) I linked at line 36.